plugins/idamagic/main_interface/__init__.py

```diff
@@ -84,6 +86,50 @@ def __init__(
         if not autoinst:
             self.parent.parent().parent().setSizes([1200, 1])
 
+    def check_ida_version(self):
+        """
+        Check if IDA version is 8.x.
+
+        Returns:
+            bool: True if version is 8.x, False otherwise
+        """
+        try:
+            # version = ida_kernwin.get_kernel_version()
+            version = "9.1"
+            major_version = int(version.split('.')[0])
+            logger.debug(f"Detected IDA version: {version} (Major: {major_version})")
+
+            if major_version != 8:
+                error_msg = (
+                    f"The Unknown Cyber plugin requires IDA version 8.x.\n"
+                    f"Detected version: {version}\n"
+                    f"Plugin loading with some features disabled, others may not work as intended."
+                )
+                QtWidgets.QMessageBox.critical(None,
+                    "IDA Version Error",
+                    error_msg
+                )
+                logger.error(
+                    f"Incompatible IDA version for use with Unknown Cyber plugin: {version}"
+                )
+                return False
+
+            return True
+
+        except Exception as e:
+            error_msg = f"Failed to check IDA version: {str(e)}"
+            logger.error(error_msg)
+            QtWidgets.QMessageBox.critical(None,
+                "IDA Version Error",
+                error_msg
+            )
+            return False
+
+    def disable_versioned_features(self):
+        """Disables features known to only work with IDA 7/8."""
```

Comment:
Is this meant to be empty?

Reply:
Good catch. This is leftover from a different approach I took. Removing